src/elements/content-sidebar/SidebarContent.js

```diff
@@ -29,7 +39,7 @@ const SidebarContent = ({ actions, children, className, title, elementId, sideba
             role="tabpanel"
             {...rest}
         >
-            <div className="bcs-content-header">
+            <div className={classNames('bcs-content-header', { 'bsc-activity-thread': hasReplies })}>
```

Comment:
typo:
```suggestion
            <div className={classNames('bcs-content-header', { 'bcs-activity-thread': hasReplies })}>
```
and in SidebarContent.scss

Reply:
and I think we'd want to use SUIT naming for the new class.

although the other class names in this file are unexpected (`bcs-content` and `bcs-title`) so I wonder if they're using some shared names

Reply:
I named it `bcs-activity-thread` to indicate that it's about Activity Thread and not any thread. We change styling a bit in regard of other sidebar content and wanted to have it behind `hasReplies` (when true this indicates that this is Threaded Replies).